client/signup/steps/site-or-domain/index.jsx

```diff
@@ -171,6 +182,32 @@ class SiteOrDomain extends Component {
 	};
 
 	render() {
+		const { translate } = this.props;
+		const productsLoaded = ! isEmpty( this.props.productsList );
```

Comment:
You seem to be checking this a lot in this file - maybe it should a selector of its own? Then you'd have it in props and would not have to create it in four different places?